### ISSUE TITLE
Partial time progression in MP

### ISSUE THREAD

**Issue by CalligoMiles** [anegostudios/VintageStory-Issues#746]
I think it would be neat if you could specify a part of the day where time does not progress if there's nobody online.

Because while it's neat to get back to a full harvest or duck out of a heavy storm as a newbie on an older server, it's frustrating to just... miss entire seasons because you need to sleep. And a good solution for this would be being able to specify that time stops between, say, midnight and 8AM.

**Comment by copygirl** [comment]
I thought time already doesn't progress if nobody is online, or am I mistaken?

Or are you suggesting a setting for time to progress (at certain times) despite nobody being online?

**Comment by CalligoMiles** [comment]
Yes. You can toggle it right now and it's great to have _some_ offline time progress... but our admin isn't the most mindful of people and certainly shouldn't have to do it every evening and morning manually.
